data_warehousing_hw_1_new_Kristin.py

```diff
@@ -0,0 +1,57 @@
+import pandas as pd
+employees = pd.read_csv("/Users/kristin.lomicka/Documents/BGSE_Courses/Data_Warehousing/warehousing-operational-databases/homework/extracts/employees.csv")
```

Comment:
No absolute path!!! Relative path!! Your code lives with the data, or your code knows where to get the data, but the same code should work on other computers!